src/components/organims/privacy/OffboardingFail.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { StaticQuery, graphql, Link } from "gatsby";
+import { css } from 'emotion';
+import { rem } from 'polished';
+
+import { mqMin } from '../../../styles/breackpoint';
+import * as font from '../../../styles/fonts';
+
+import SectionMinimal from '../../molecules/SectionMinimal';
+import Title from '../../atoms/Title';
+import Button from '../../atoms/Button';
+import Content from '../../molecules/Content';
+
+class OffboardingFail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.navigate = props.navigate;
+    this.actualRender = this.actualRender.bind(this);
+  }
+
+  contactUs() {
+    window.location.href = "mailto:privacy@getstation.com";
```

Comment:
Shouldn't be `window.open('mailto:privacy@getstation.com')`? Because otherwise the current page is supposed to navigate.

Reply:
Mix of double and singlequote :)